test/helpers/DaoBuilder.sol

```diff
@@ -178,8 +184,10 @@ contract DaoBuilder is Test {
             taikoL1 = new TaikoL1Mock();
         } else if (taikoL1Status == TaikoL1Status.Paused) {
             taikoL1 = new TaikoL1PausedMock();
-        } else {
+        } else if (taikoL1Status == TaikoL1Status.OutOfSync) {
             taikoL1 = new TaikoL1WithOldLastBlock();
+        } else {
+            taikoL1 = TaikoL1(address(new TaikoL1Incompatible()));
```

Comment:
The builder always returns a contract even if the contract is incompatible, I believe try/catch controls for this (IIRC it's just syntactic sugar for `address.call(data);` under the hood) but do we test for the case when it reaches into the address space where there is NO contract?

Reply:
@jordaniza not sure I follow

The builder always returns a builder, but the highlighted line reproduces the case where an external dependency (TaikoL1) for some reason becomes corrupt or unusable. 
This could create a deadlock: the DAO controles the protocol but the DAO depends on a broken protocol contract.

In this test line we are manually bypassing the compiler check against the TaikoL1 interface. `TaikoL1Incompatible` is an empty contract on purpose, to test what happens when calling `t1.paused()` fails. 

Testing for `taikoL1` being empty should rather be done on the initialize function. Pushed a commit with that

Reply:
nice, that makes perfect sense thank you